lib/blocks.php

```diff
@@ -412,3 +412,36 @@ function gutenberg_block_has_support( $block_type, $feature, $default = false )
 
 	return true === $block_support || is_array( $block_support );
 }
+
+/**
+ * Updates the shape of supports for declaring fontSize and lineHeight.
+ *
+ * @param array $metadata Metadata for registering a block type.
+ * @return array          Metadata for registering a block type with the supports shape updated.
+ */
+function gutenberg_migrate_old_typography_shape( $metadata ) {
+	if ( isset( $metadata['supports'] ) ) {
+		$font_size_support   = _wp_array_get( $metadata['supports'], array( 'fontSize' ), null );
+		$line_height_support = _wp_array_get( $metadata['supports'], array( 'lineHeight' ), null );
+		if ( null !== $font_size_support ) {
+			trigger_error(
```

Comment:
This is what I see using lates5.8-alpha-51034 (it doesn't happen on 5.7):

https://user-images.githubusercontent.com/583546/119811731-b77bd180-bee7-11eb-8ee5-a25c002d6fd5.mp4

Notice that I can't navigate anywhere :sweat_smile: Given the high usage of `fontSize` and `lineHeight` (which means a lot of messages) and that this migration makes everything work, is there a way to tone down the notifications a bit?

Reply:
That was happening because the core block.json files are not yet updated and are also processed by the filter. For now, I added a check to ignore core files until we update the core.